src/Common/hooks/useNotificationSubscribe.ts

```diff
@@ -0,0 +1,138 @@
+import { useDispatch } from "react-redux";
+import {
+  getUserPnconfig,
+  updateUserPnconfig,
+  getPublicKey,
+} from "../../Redux/actions";
```

Comment:
replace to use useQuery/request. Newer codebase should not contain calls with redux.